test/active_model_serializers/test/serializer_test.rb

```diff
@@ -17,6 +23,11 @@ def render_some_text
 
       tests MyController
 
+      def test_supports_specifying_serializers_with_a_multi_serializer_class
+        get :render_multi_using_serializer
+        assert_serializer ProfileSerializer
```

Comment:
`assert_each_serializer` may be cool, yeah.
My test is dumb on purpose, and sure if I declare it explicitely 5 lines earlier it looks silly. But in our process, we usually don't have code yet when writing tests. And anyway we test it, declared explicitly or not.

(anyway, the doc was misleading to me, do you think it should be edited ?)

Thanks